docs/guides/0-web-verification.md

```diff
@@ -0,0 +1,217 @@
+
+## **Overview**
+
+The W3C DC API integration allows your app to interact with web-based verifiers via the Credential Manager API, supporting secure and privacy-preserving credential presentment flows. With this, users can share credentials directly with browsers that support the API. To implement this using Multipaz SDK, these steps are required:
+
+* Implementing a new `CredmanActivity` to handle credential requests.
+* Registering privileged user agents (browsers) via a `privilegedUserAgents.json` file.
+* Updating the app initialization to support exporting credentials through the API.
+
+## **How It Works**
+
+When a user visits a web verifier (like [verifier.multipaz.org](https://verifier.multipaz.org)), here's what happens:
+
+1. **Browser Request**: The verifier website uses the W3C Digital Credentials API to request credentials
+2. **Android Response**: Your app (as a credential provider) receives the request and shows the credential selection UI
+3. **Server Verification**: The verifier server processes and verifies the returned credentials
+4. **Result Display**: The browser displays the verified credential information
+
+The verifier server handles the complex cryptographic verification, while your Android app focuses on secure credential presentation.
+
+## **Implementation Steps**
+
+### **1. Add CredmanActivity**
+
+Create a new activity extending `CredentialManagerPresentmentActivity `provided by Multipaz. This activity is launched when a browser requests credentials via the W3C DC API.
+
+```
+package org.multipaz.getstarted
+
+import androidx.compose.runtime.Composable
+import multipazgettingstartedsample.composeapp.generated.resources.Res
+import org.multipaz.compose.digitalcredentials.CredentialManagerPresentmentActivity
+
+class CredmanActivity : CredentialManagerPresentmentActivity() {
+    @Composable
+    override fun ApplicationTheme(content: @Composable (() -> Unit)) {
+        App()
+        content()
+    }
+
+    override suspend fun getSettings(): Settings {
+        val app = App.getInstance()
+        app.init()
+        return Settings(
+            appName = app.appName,
+            appIcon = app.appIcon,
+            promptModel = App.promptModel,
+            documentTypeRepository = app.documentTypeRepository,
+            presentmentSource = app.presentmentSource,
+            privilegedAllowList = Res.readBytes("files/privilegedUserAgents.json").decodeToString()
+        )
+    }
+}
+```
+
+Refer to **[**the CredmanActivity file**](https://github.com/openwallet-foundation/multipaz-samples/blob/af7f2adea05fd2ca531cdc50824e556b88b770fd/MultipazGettingStartedSample/composeApp/src/androidMain/kotlin/org/multipaz/getstarted/CredmanActivity.kt)** for the complete example.
+
+### **2. Update AndroidManifest.xml**
+
+Register `CredmanActivity` in your manifest and declare intent filters for the Credential Manager API actions.
+
+```
+<activity
+    android:name=".CredmanActivity"
```

Comment:
Do we need to show the full code here, or would just the intent filter be enough? If the intent filter is the main point, we could simplify the snippets to remove some of the extra noise.